packages/story-editor/src/utils/copyPaste.tsx

```diff
@@ -30,16 +33,30 @@ import generateGroupName from './generateGroupName';
 
 const DOUBLE_DASH_ESCAPE = '_DOUBLEDASH_';
 
+interface ProcessPastedElementsReturn {
+  animations: StoryAnimation[];
+  elements: Element[];
+  groups?: Groups;
+}
+
+interface Payload {
+  sentinel?: string;
```

Comment:
Dafuq is _sentinel_? Isn't that those robots from Matrix?

![sentinel](https://static.wikia.nocookie.net/robotsupremacy/images/0/08/Squiddy.jpg/revision/latest?cb=20120512111003)

Reply:
Yep, it has its similarities.